### PR TITLE
Fix: App directly minimizes on back pressed in Find in page

### LINKED ISSUE #672: App directly minimizes on back pressed in Find in page
### Environment
  * version of the software you use : 2.3 build 55
  * device / operating system : Motorola E3 Power, Android version 6.0
  
### The Bug:

  * What do you obtain? A copy of the error message or a screenshot is always useful.

  * When does this occur? 

    Upon back pressed when the `Find in page` functionality is active, the app directly minimizes instead of just closing the Search toolbar.

#### Steps to reproduce:

  1. Click on `Find in page` feature
  2. Minimize the keyboard
  3. Press the back button again.
  
  * What should be the behaviour from your point of view? How do you expect the service to work?

    Just the `ActionMode` which displays the `Find in page` functionality should be closed.

### PR DESCRIPTION
**Pull request by brijeshshah13**
Fixes #672 

Changes: Implemented the back button action using `onKeyDown()` to close the `ActionMode` of `Find in page`. 

Screenshots/GIF for the change: Attached in the issue itself.

### CODE REVIEW

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 1]
```diff
@@ -1166,12 +1166,11 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
             getCurrentWebView().goBack();
           } else if (isFullscreenOpened) {
             closeFullScreen();
+          } else if (compatCallback.mIsActive) {
```
Comment (hjoshi123) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Hey could you directly make it go back to the page of content? I have to press back twice to get back to the page?

Comment (brijeshshah13) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: If implemented, whenever the user will type in something to search and go back to viewing its presence in the content, he/she won't be able to, as the `Find in page` functionality as already been closed by just a single back action. Thus, I think the current flow is correct. Please think it over.

Comment (hjoshi123) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Cool. No issues. I didn't think of that

**Comment by brijeshshah13** [comment]
@mhutti1 Please review.

**Comment by brijeshshah13** [comment]
@mhutti1 The PR is pretty straight forward. Please review and merge.

**Pull request merged**